lib/encoding/json.toit

```diff
@@ -40,6 +41,37 @@ Utf-8 encoding is used for strings.
 encode obj -> ByteArray:
   return encode obj: throw "INVALID_JSON_OBJECT"
 
+/**
```

Comment:
This documentation should be on `encode-stream o --writer`.
The version with a block|lambda converter should be variants of the one without.